### ISSUE TITLE
Start and end marker im leaflet

### ISSUE THREAD

**Issue by labneu** [bobosch/ods_osm#35]
how can i disable the start and end Marker Icon in leaflet

**Comment by TomQv** [comment]
Yes, I also ran in that issue now. I don't want these start and end markers in the map.  Workaround for me is to change then to a small, empty bitmap in typo3conf/ext/ods_osm/Resources/Public/JavaScript/Leaflet/leaflet-gpx/ 
But I belive, with the next update of ods_osm the originals will be back
